src/app.rs

```diff
@@ -359,16 +454,42 @@ impl eframe::App for RustyAutoClickerApp {
         let ms: u64 = parse_string_to_u64(self.ms_str.clone());
         // println!("{} hr {} min {} sec {} ms", &hr, min, sec, ms);
 
+        // Parse movement Strings to u64
+        let movement_sec: u64 = parse_string_to_u64(self.movement_sec_str.clone());
+        let movement_ms: u64 = parse_string_to_u64(self.movement_ms_str.clone());
+
+        // Calculate movement delay
+        let movement_delay_in_ms: u64 = (movement_sec * 1000u64) + movement_ms;
+
         // Parse click amount String to u64
-        let mut click_amount: u64 = 0u64;
-        if !self.click_amount_str.is_empty() {
-            click_amount = self.click_amount_str.parse().unwrap();
-        }
+        let click_amount: u64 = parse_string_to_u64(self.click_amount_str.clone());
 
         // Parse mouse coordinates Strings to f64
         let click_x: f64 = parse_string_to_f64(self.click_x_str.clone());
         let click_y: f64 = parse_string_to_f64(self.click_y_str.clone());
 
+        // Toggle autoclicking
+        if self.key_autoclick.is_some() && keys.contains(&self.key_autoclick.unwrap()) {
+            self.key_pressed_autoclick = true;
+        } else if self.key_pressed_autoclick {
+            self.key_pressed_autoclick = false;
+            if self.is_autoclicking {
+                self.is_autoclicking = false;
+            } else {
+                // Set only if app is not busy
+                if !self.is_setting_autoclick_key
+                    && !self.is_setting_coord
+                    && !self.is_setting_set_coord_key
+                    && !self.hotkey_window_open
+                {
+                    self.click_counter = 0u64;
+                    self.is_autoclicking = true;
+                    self.rng_thread = thread_rng();
+                }
+                self.last_now = Instant::now();
+            }
+        }
+
```

Comment:
Found the bug! This is old code that was replaced by what's on line 520.
It is probably my bad when I "resolved" the merge conflicts. My apologies!

Please delete these lines of code.